.env.sample

```diff
@@ -1,4 +1,8 @@
-NYPL_API_BASE_URL
-NYPL_OAUTH_URL
-NYPL_OAUTH_KEY
-NYPL_OAUTH_SECRET
\ No newline at end of file
+# GENERAL CONFIGURATION
+APP_ENV=development
+
+# API ENDPOINT
+API_URL=localhost:5000
```

Comment:
let's put a non-local API url here for easy copy-pasting in the future

Reply:
Could we make this an optional variable?  So if one isn't passed, it's derived from `APP_ENV` (localhost for dev, the qa api for qa, etc)

Reply:
that's a good idea I think (for both comments). Easy enough to do and I think that QA and dev would point to the same URL